students/student_framllat/session03/slicer.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
```

Comment:
the idea here is to use slicing as much as possible -- which works with all sequences.

So there should be no need to convert to a list.